### DISCUSSION TITLE
Error en el Lab 02.03 Tak3 (Fill in missing data=

### DISCUSSION THREAD

**Discussion by soyudea** [rramosp/ai4eng.v1#229] [02 - PYTHON]
El codigo me genera el dataframe pero dice que hay ambiguedad, **no encuentro el problema**

def fillna(df):
  df = df.copy()
  precio =d["price"]
  media=precio.mean()
  std = np.std(precio)
  ind = df.index
  malos= d['price'].isna().sum()
  dat = np.random.normal(media, std, size=malos)
  k=0
  for i in range(len(d)):
       if np.isnan(df.loc[ind[i],"price"])==True:
            #j= np.random.randint(0, len(dat))
            df.loc[ind[i],"price"] = round(dat[k],8)
            k +=1
 
  result = df   
  return result

ValueError: The truth value of a Series is ambiguous. Use a.empty, a.bool(), a.item(), a.any() or a.all(). (set session.debug=True for tracebacks)

**Comment by imEag** [comment]
Perfecto! Ya se solucionó en clase, sin embargo, si quieres ver otra solución mira este post https://github.com/rramosp/ai4eng.v1/discussions/226
